src/views/dialogs/OnboardingDialog/ChooseWallet.tsx

```diff
@@ -52,11 +49,16 @@ export const ChooseWallet = () => {
           <Styled.WalletButton
             action={ButtonAction.Base}
             key={walletType}
-            onClick={() => selectWalletType(walletType)}
+            onClick={() => {
+              if (walletType === WalletType.Privy) {
+                setIsOpen(false);
+              }
```

Comment:
nit: I think we should move this logic to `OnboardingDialog` as a `onChooseWallet` callback

Reply:
I think the nit just means passing in:
```
onClick={() => {
  onChooseWallet();
  selectWalletType(walletType);
}}
```
and setting it like this on OnboardingDialog:
```
<ChooseWallet onChooseWallet={() => {
  if (walletType === WalletType.Privy) {
    setIsOpen(false);
  }
}} />